src/app/check/middleware/session-check-middleware.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  sessionCheckMiddleware: async (req, res, next) => {
+    if (!req.session?.tokenId || !req.session?.authParams) {
```

Comment:
What do you think about adding a `logger.error` in here? Might be helpful for debugging or reporting on the errors?

I generally feel that we're not logging enough in our frontends to make debugging easy